buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidLibraryRuleComposer.groovy

```diff
@@ -46,6 +49,22 @@ final class AndroidLibraryRuleComposer extends AndroidBuckRuleComposer {
             testTargets.add(":${test(target)}")
         }
 
+        List<String> extraKotlincArgs = []
+        if (target.hasKotlinAndroidExtensions) {
+            String options = ""
+            target.getResDirs().each {  String dir ->
+                // :root:module -> /root/module/
+                String module = target.project.path.replace(":", "/") + "/"
+                // src/main/res -> main
+                String variant = dir.substring(dir.indexOf("/") + 1, dir.lastIndexOf("/"))
```

Comment:
I'll double check, I think it was defaulting to `debug` in both cases. I need it to be both `main`and `debug`.